src/routes/+page.svelte

```diff
@@ -14,10 +16,16 @@
 	import Logo from '$lib/components/Logo.svelte';
 	import Footer from '$lib/components/Footer.svelte';
 
-	export let data: PageData;
+	interface Props {
+		data: PageData;
+	}
+
+	let { data }: Props = $props();
 
-	let { supabase, socialThumbImgURL } = data;
-	$: ({ supabase } = data);
+	let { supabase, socialThumbImgURL } = $state(data);
+	run(() => {
```

Comment:
same here?

Reply:
The entire `run` block and import should be removed and we can do
```js
let { socialThumbImgURL } = $derived(data);
```
I am unsure why `supabase` is being deconstructed as it is unused throughout the file. If we are relying on a side effect of assigning to the variable, that should be an `$effect` rune.